src/__tests__/__snapshots__/main-test.js.snap

```diff
@@ -81,6 +82,7 @@ Object {
       "required": true,
       "type": Object {
         "name": "shape",
+        "required": true,
```

Comment:
Can you elaborate in which example exactly it is not adding the required field? Because in this example it is now twice, because we already add the required field in the prop object. see line 82.